src/Model/GroundWaterTransport/gwt1src1.f90

```diff
@@ -270,17 +260,17 @@ subroutine src_df_obs(this)
     call this%obs%StoreObsType('to-mvr', .true., indx)
     this%obs%obsData(indx)%ProcessIdPtr => DefaultObsIdProcessor
     !
-    ! -- return
+    ! -- Return
     return
   end subroutine src_df_obs
 
-  ! -- Procedure related to time series
-
+  !> @brief Procedure related to time series
+  !!
+  !! Assign tsLink%Text appropriately for all time series in use by package.
+  !! In the SRC package only the SENERRATE variable can be controlled by time
```

Comment:
RATE?

Reply:
Restoring `SENERRATE` -> `SMASSRATE`.  Regrettable that so many GWE related modifications crept into the docstrings...thanks for catching.  Restored with 8382809